python/nav/web/jwtgen.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime, timedelta
+from typing import Dict, Any
+import jwt
+from nav.jwtconf import JWTConf
+
+
+def generate_access_token(token_data: Dict[str, Any] = {}) -> str:
+    """Generates and returns an access token in JWT format.
+    Will use `token_data` as a basis for the new token,
+    but certain claims will be overridden.
+    """
+    return _generate_token(token_data, JWTConf.ACCESS_EXPIRE_DELTA, "access_token")
+
+
+def generate_refresh_token(token_data: Dict[str, Any] = {}) -> str:
+    """Generates and returns a refresh token in JWT format.
+    Will use `token_data` as a basis for the new token,
+    but certain claims will be overridden.
+    """
+    return _generate_token(token_data, JWTConf.REFRESH_EXPIRE_DELTA, "refresh_token")
+
+
+def _generate_token(
+    token_data: Dict[str, Any], expiry_delta: timedelta, token_type: str
+) -> str:
+    """Generates and returns a token in JWT format. Will use `token_data` as a basis
+    for the new token, but certain claims will be overridden
+    """
+    new_token = dict(token_data)
+    now = datetime.now()
```

Comment:
I know NAV has been lazy with naive timestamps in most places, but in a security context like this,  we should really use a timezone-aware timestamp.  AFAICT, JWT operates on UTC timestamps, but how does a timezone-naive datetime object know how to output a UTC-based timestamp?

Reply:
Good point, imma fix that